samples/snippets/src/main/java/aiplatform/GetFeatureSample.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *
+ * Get feature details. See 
+ * https://cloud.google.com/vertex-ai/docs/featurestore/setup before running 
+ * the code snippet
+ */
+
+package aiplatform;
+
+// [START aiplatform_get_feature_sample]
+
+import com.google.cloud.aiplatform.v1.Feature;
+import com.google.cloud.aiplatform.v1.FeatureName;
+import com.google.cloud.aiplatform.v1.FeaturestoreServiceClient;
+import com.google.cloud.aiplatform.v1.FeaturestoreServiceSettings;
+import com.google.cloud.aiplatform.v1.GetFeatureRequest;
+import java.io.IOException;
+
+public class GetFeatureSample {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String project = "YOUR_PROJECT_ID";
+    String featurestoreId = "YOUR_FEATURESTORE_ID";
+    String entityTypeId = "YOUR_ENTITY_TYPE_ID";
+    String featureId = "YOUR_FEATURE_ID";
+    String location = "us-central1";
+    String endpoint = "us-central1-aiplatform.googleapis.com:443";
+
+    getFeatureSample(project, featurestoreId, entityTypeId, featureId, location, endpoint);
+  }
+
+  static void getFeatureSample(String project, String featurestoreId, String entityTypeId,
+      String featureId, String location, String endpoint) throws IOException {
+
+    FeaturestoreServiceSettings featurestoreServiceSettings =
+        FeaturestoreServiceSettings.newBuilder().setEndpoint(endpoint).build();
+
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (FeaturestoreServiceClient featurestoreServiceClient =
```

Comment:
should we add the close to examples as well ?

Reply:
In other samples of GitHub close is not used, so we were following the same approach. 
Should we add it for all the feature samples?

Reply:
Yeah I see it as a plus to have it be there.

Reply:
added for all feature API's